src/fiber/graph.rs

```diff
@@ -1612,6 +1614,10 @@ impl PaymentSession {
         !self.request.router.is_empty()
     }
 
+    pub fn first_hop_channel_outpoint(&self) -> Option<OutPoint> {
```

Comment:
suggest to add an out point parameter since the only use case is to comparison:
```rust
    pub fn first_hop_channel_outpoint_eq(&self, out_point: &OutPoint) -> bool {
        self.route.nodes.first().map(|x| x.channel_outpoint.eq(out_point)).unwrap_or_default()
    }
```

Reply:
fixed.